tests/pages/firefox/accounts.py

```diff
@@ -2,24 +2,35 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
+from selenium.common.exceptions import TimeoutException
 from selenium.webdriver.common.by import By
+from selenium.webdriver.support import expected_conditions as expected
+from selenium.webdriver.support.ui import WebDriverWait as Wait
 
 from pages.firefox.base import FirefoxBasePage
-from pages.regions.download_button import DownloadButton
 
 
 class FirefoxAccountsPage(FirefoxBasePage):
 
     URL_TEMPLATE = '/{locale}/firefox/accounts/'
 
-    _download_button_locator = (By.ID, 'features-hero-download')
+    _download_buttons_locator = (By.CLASS_NAME, 'features-download')
```

Comment:
I believe right now the test is only matching the first instance of the download button it finds in the page. We should probably add an assert for each instance, so if one breaks we can know more easily which one has an issue.